Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM debian
```

Comment:
Unsure if foundry can build on alpine, but `debian` is 124MB while `alpine` is 5.58MB. Definitely prefer building based on alpine if possible

Reply:
I've attempted to install it on alpine, but the build keeps failing due to dependency issues. Please feel free to try it on your own.